docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -0,0 +1,1382 @@
+RETURN count(*);
+----
+
+Copy and paste the contents of the script into a file called `import.cypher`.
+
+Next, copy the import cypher script and CSV files into Neo4j's import folder:
+
+[source,bash]
+----
+podman cp \
+    target/*-native-image-source-jar/reports \
+    testneo4j:/var/lib/neo4j/import
+
+podman cp import.cypher testneo4j:/var/lib/neo4j
+----
+
+After copying all the files, invoke the import script:
+
+[source,bash]
+----
+podman exec testneo4j bin/cypher-shell -u neo4j -p ${NEO_PASS} -f import.cypher
+----
+
+Once the import completes (shouldn't take more than a couple of minutes), go to the Neo4j browser,
+and you'll be able to observe a small summary of the data in the graph:
+
+image:native-reference-neo4j-db-info.png[Neo4j database information after import]
+
+The data above shows that there are ~60000 methods, and just over ~200000 edges between them.
+The Quarkus application demonstrated here is very basic, so there’s not a lot we can explore, but here are some example queries you can run to explore the graph in more detail.
+Typically, you’d start by looking for a given method:
+
+[source,cypher]
+----
+match (m:Method) where m.name = "hello" return *
+----
+
+From there, you can narrow down to a given method on a specific type:
+
+[source,cypher]
+----
+match (m:Method) where m.name = "hello" and m.type =~ ".*GreetingResource" return *
+----
+
+Once you’ve located the node for the specific method you’re after, a typical question you’d want to get an answer for is:
+why does this method get included in the call tree?
+To do that, start from the method and look for incoming connections at a given depth,
+starting from the end method.
+For example, methods that directly call a method can be located via:
+
+[source,cypher]
+----
+match (m:Method) <- [*1..1] - (o) where m.name = "hello" return *
+----
+
+Then you can look for direct calls at depth of 2,
+so you’d search for methods that call methods that call into the target method:
+
+[source,cypher]
+----
+match (m:Method) <- [*1..2] - (o) where m.name = "hello" return *
+----
+
+You can continue going up layers,
+but unfortunately if you reach a depth with too many nodes,
+the Neo4j browser will be unable to visualize them all.
+When that happens, you can alternatively run the queries directly against the cypher shell:
+
+[source,bash]
+----
+podman exec testneo4j bin/cypher-shell -u neo4j -p ${NEO_PASS} \
+  "match (m:Method) <- [*1..10] - (o) where m.name = 'hello' return *"
+----
+
+=== Used Packages/Classes/Methods Reports
+
+`used_packages`, `used_classes` and `used_methods` text file reports come in handy when comparing different versions of the application,
+e.g. why does the image take longer to build? Or why is the image bigger now?
+
+=== Further Reports
+
+Mandrel can produce further reports beyond the ones that are enabled with the `-Dquarkus.native.enable-reports` option.
+These are called expert options and you can learn more about them typing:
```

Comment:
Maybe have a small section describing what information these expert reports provide?

Reply:
Well, typing the above gives you information about those, e.g. [sample output](https://gist.github.com/galderz/8fbab86d55210206994eaeeabf03343c). I didn't want to more information on those because they could change in next GraalVM version, or they could disappear. We don't normally use them unless we're looking for something very specific.